content/flux/v0.x/data-types/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Work with Flux data types
+description: >
+  ...
+menu:
+  flux_0_x:
+    name: Work with data types
+weight: 4
+---
+
+A Flux **type** defines the set of values and operations possible on those values.
+Data types are never explicitly declared as part of the Flux syntax except as
+part of a [builtin statement](/flux/v0.x/spec/system-built-ins/).
+Types are inferred from how a value is used.
+
+Flux is a **statically typed** so variable types are inferred at compile time.
+Static typing associates types with variables, not with values.
```

Comment:
`Static typing associates every variable with a type during compilation.`

Reply:
Thanks for the suggestion. This wording is hard. I'm worried your suggestion loses some of the original meaning. What about:

> Static typing associates a type with a variable, not a variable's value.

For example, if I were to say `foo = "bar"`, to the Flux compiler, this means `foo` is a string. The type isn't associated with `"bar"`, but the type of `foo` is inferred from `"bar"`.

Reply:
Hm. I think the issue I'm having is that this sentence on line 17 seems to be addressed at a particular confusion that the reader might have, but I think it might actually be more confusing to address this.

Reply:
Maybe we need to describe type inference in some more general terms.

"Static typing looks at the kinds (or "sorts?") or values assigned to variables and the way those variables are used to deduce a type for every variable."